docs/release-notes/NuGet-5.8.md

```diff
@@ -0,0 +1,170 @@
+---
+title: NuGet 5.8 Release Notes
+description: Release notes for NuGet 5.8 including new features, bug fixes, and DCRs.
+author: dominofire
+ms.author: feaguila
+ms.date: 11/9/2020
+ms.topic: conceptual
+---
+
+# NuGet 5.8 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**5.8**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.8](https://visualstudio.microsoft.com/downloads/) | [<SDKVersion>](https://dotnet.microsoft.com/download/dotnet-core/5.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio <VSYear> with.NET Core workload
+  
+> [!NOTE]
+> For maximum compatibility of NuGet 5.8, it is recommended to use Visual Studio 16.8, MSBuild 16.8, and .NET 5.0.
+
+
+## Summary: What's New in 5.8
+
+* Package Manager sources use remote types from server - [#9608](https://github.com/NuGet/Home/issues/9608)
+
+* Search CLI Tool for NuGet.exe - [#9704](https://github.com/NuGet/Home/issues/9704)
+
+* Add a TypeConverter for the SemanticVersion class - [#9125](https://github.com/NuGet/Home/issues/9125)
```

Comment:
this is low relevancy too. Should be in the log list, not top paragraph. 

We should look into addressing the tool/change the tags correctly.